nixos/modules/services/games/pterodactyl-wings.nix

```
@@ -0,0 +1,214 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.services.pterodactyl-wings;
+  settingsFormat = pkgs.formats.yaml {};
+in
+{
+  options.services.pterodactyl-wings = with lib; with lib.types; {
+    enable = mkEnableOption "the Pterodactyl wings server manager";
```

```suggestion
    enable = mkEnableOption "the Pterodactyl wings server manager";

```

Adding some new lines makes it easier to understand the code for developers and reviewers.